apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RdaServerApp.java

```diff
@@ -39,7 +39,12 @@ public static void main(String[] args) throws Exception {
     final Config config = new Config(args);
     LOGGER.info("Starting server on port {}.", config.getPort());
     Server server =
-        RdaServer.startLocal(config.getPort(), config::createFissClaims, config::createMcsClaims);
+        RdaServer.startLocal(
```

Comment:
is this PR just building off the builder changes introduced in the API auth PR?

Reply:
Yes it is.  Now that the API auth PR has been merged I've been able to merge that into here formally and deal with the few conflicts.

Reply:
Nice outcome of that is that this PR has a much smaller changed files list.